### PR TITLE
Custom KTON Metadata

### PR DESCRIPTION
**Pull request by AsceticBear**

### CODE REVIEW

File: frame/dvm/evm/precompiles/kton/src/lib.rs [hunk 1]
```diff
@@ -40,17 +40,25 @@ use frame_support::{
 };
 use sp_core::{H160, U256};
 
-const TOKEN_NAME: &str = "Wrapped KTON";
-const TOKEN_SYMBOL: &str = "WKTON";
-const TOKEN_DECIMAL: u8 = 18;
+/// Metadata of an ERC20 token.
+pub trait ERC20Metadata {
```
Comment (AurevoirXavier) [inline frame/dvm/evm/precompiles/kton/src/lib.rs:44]: Recommend using `Erc20Metadata`

Comment (AsceticBear) [reply frame/dvm/evm/precompiles/kton/src/lib.rs:44]: I was torn between those two several times. :smile:

File: frame/dvm/evm/precompiles/kton/src/lib.rs [hunk 2]
```diff
@@ -40,17 +40,25 @@ use frame_support::{
 };
 use sp_core::{H160, U256};
 
-const TOKEN_NAME: &str = "Wrapped KTON";
-const TOKEN_SYMBOL: &str = "WKTON";
-const TOKEN_DECIMAL: u8 = 18;
+/// Metadata of an ERC20 token.
+pub trait Erc20Metadata {
```
Comment (AurevoirXavier) [inline frame/dvm/evm/precompiles/kton/src/lib.rs:44]: Should we put this in kton precompile?
It looks like a more generic property.

Comment (AsceticBear) [reply frame/dvm/evm/precompiles/kton/src/lib.rs:44]: Let's keep it here for now, no other modules will be used it at the moment.

**Review by HackFisher — APPROVED** [review]

**Pull request merged**